Week-2/style2_2.css

```diff
@@ -0,0 +1,346 @@
+*{
```

Comment:
can break larger css files in multiple smaller files, large files are not easy to review.